test/cloudant.test.js

```diff
@@ -498,20 +505,6 @@ describe('cloudant constructor', function() {
     done();
   });
 
-  it('should give 401 error for wrong creds', function(done) {
```

Comment:
a dup test case of https://github.com/strongloop/loopback-connector-cloudant/blob/master/test/cloudant.connection.test.js